utility/src/main/java/org/oppia/util/data/UrlImageParser.kt

```diff
@@ -0,0 +1,61 @@
+package org.oppia.util.data
+
+import android.content.Context
+import android.graphics.Bitmap
+import android.graphics.Canvas
+import android.graphics.Rect
+import android.graphics.drawable.BitmapDrawable
+import android.graphics.drawable.Drawable
+import android.graphics.drawable.LevelListDrawable
+import android.text.Html
+import android.widget.TextView
+import com.bumptech.glide.Glide
+import com.bumptech.glide.request.target.SimpleTarget
+import com.bumptech.glide.request.transition.Transition
+import java.net.URL
+
+/** UrlImage Parser for android TextView to load Html Image tag. */
+class UrlImageParser(internal var tvContents: TextView, internal var context: Context) : Html.ImageGetter {
+
+  var targets: ArrayList<BitmapTarget>? = null
```

Comment:
Why do we need this list? I'm concerned that it's potentially leaking these bitmap targets.

Reply:
It was not used anywhere so removed it